src/frontend/web_application/src/scenes/DiscussionList/components/DiscussionItemActionsContainer/presenter.jsx

```diff
@@ -57,7 +57,11 @@ class DiscussionItemActionsContainer extends Component {
 
     return (
       <div
-        className={classnames('m-discussion-item-actions-container__actions', { 'm-discussion-item-actions-container--active__actions': this.state.isActive })}
+        className={classnames(
+          'm-discussion-item-actions-container__actions',
+          { 'm-discussion-item-actions-container--active__actions': this.state.isActive },
+          'show-for-medium'
```

Comment:
I think we should completely disable `DiscussionItemActionsContainer` in parent components and open an issue for @Lauth0. 
We need a good solution that only works for all screen resolutions. What do think ?

Reply:
I finally restored it, and open issue https://github.com/CaliOpen/Caliopen/issues/464